core/storage/ipfs/impl/datastore_leveldb.cpp

```diff
@@ -41,6 +41,7 @@ namespace fc::storage::ipfs {
   }
 
   outcome::result<void> LeveldbDatastore::set(const CID &key, Value value) {
+    // maybe check value hash matches cid
```

Comment:
As this was discussed on a meeting, just a reminder to remove this comment. Or make a TODO.